tests/alloc.rs

```diff
@@ -0,0 +1,89 @@
+#![cfg(feature = "alloc")]
+
+use classic_mceliece_rust::{decapsulate_boxed, encapsulate_boxed, keypair, keypair_boxed};
+use classic_mceliece_rust::{CRYPTO_PUBLICKEYBYTES, CRYPTO_SECRETKEYBYTES};
+use std::thread;
+
+/// The smallest amount of stack needed to reliably run the KEM from this library.
+const MIN_STACK_SIZE_FOR_THIS_CRATE: usize = (CRYPTO_PUBLICKEYBYTES as f32 * 1.65) as usize;
```

Comment:
So.. This library needs a huge amount of stack, even when you put the keys on the heap. It requires *less* stack when you put the keys on the heap, but still a lot.

This test guards for stack usage regressions.